bin/sync/article_updater.rb

```diff
@@ -162,6 +162,12 @@ def generate_markdown(article_data, slug, description)
       "slug" => slug
     }.merge(metatags_image)
 
+    article_data["body_markdown"] = prepare_youtube_links(article_data["body_markdown"])
+
     "#{article_hash.to_yaml(line_width: -1)}---\n#{article_data["body_markdown"]}"
   end
+
+  def prepare_youtube_links(markdown)
+    markdown.gsub(/\{\{\< youtube ([a-zA-Z0-9_-]+) \>\}\}/, '{{< youtube id=\1 loading=lazy start=30 >}}')
```

Comment:
Yeah, lets remove the  start param and update the post: https://jetthoughts.com/blog/how-find-your-next-startup-idea-lessons-from-y-combinator-management/